tests/js/list-tests.js

```diff
@@ -1601,4 +1601,61 @@ module.exports = {
         });
         realm.close();
     },
+
+    testClassObjectCreation: function() {
+        class TodoItem extends Realm.Object {
+            constructor(description) {
+                super()
+                this.id = new ObjectId()
+                this.description = description;
+                this.done = false;
+            }
+        }
+
+        TodoItem.schema =  {
```

Comment:
The tests usually put their schemas in `schemas.js`..  Is there any specific reason why these don't?

Reply:
Well I would perhaps like to have a conversation about this.  I find in tests it is best to not put such things in separate files, as they rarely are reused and it makes the reader of the test have a nicer time following what is happening.  So I guess that's more my personal style and opinion.

Reply:
I don't have a strong opinion on the subject myself..
@kneth ?

Reply:
We are using the same schemas across tests, so we avoid code duplication with `schemas.js`.

Reply:
We are going to create a new issue for this where we will address refactor the tests and write a style guide.